v03_pipeline/lib/tasks/base/base_variant_annotations_table_test.py

```diff
@@ -49,7 +51,11 @@ def setUp(self) -> None:
             ),
         )
 
-    def test_should_create_initialized_table(self) -> None:
+    @patch(
+        'v03_pipeline.lib.tasks.base.base_variant_annotations_table.UpdatedReferenceDatasetCollectionTask',
+    )
```

Comment:
I was iffy on mocking the entire output of this task but since it's already unit tested in its own test file it made sense to me to just mock the output as complete instead of mocking all of the dataset tables, the config dict, and other task dependencies when it's not really the purpose of the test.